Mathlib/Combinatorics/SimpleGraph/Basic.lean

```diff
@@ -630,6 +633,9 @@ instance [DecidableEq V] [Fintype s] : Fintype (fromEdgeSet s).edgeSet := by
 
 end FromEdgeSet
 
+theorem disjoint_left {G H : SimpleGraph V} : Disjoint G H ↔ ∀ x y, G.Adj x y → ¬H.Adj x y := by
```

Comment:
I know this follows `Set.disjoint_left`, but I've never liked that name. What about simply
```suggestion
theorem disjoint_iff {G H : SimpleGraph V} : Disjoint G H ↔ ∀ x y, G.Adj x y → ¬H.Adj x y := by
```
?

Reply:
I also don't like that name, but we need some way of distinguishing the two possible RHSs here

Reply:
I understand, but in practice do we even care about the other RHS? They are equivalent for all term-mode purposes, so it ought to make a difference when rewriting, but `Set.disjoint_right : Disjoint s t ↔ ∀ x ∈ t, x ∉ s` is a "bad" rewriting lemma because it swaps the positions of `s` and `t` within the expression